HSTracker/Utility/BoardDamage/BoardHero.swift

```diff
@@ -12,24 +12,47 @@ class BoardHero: IBoardEntity {
     private(set) var _baseAttack = 0
     private(set) var _hero: BoardCard
     private(set) var _weapon: BoardCard?
-    
-    var name: String { return _hero.name }
-    var hasWeapon: Bool { return _weapon != nil }
-    
+
+    var name: String {
+        return _hero.name
+    }
+    var hasWeapon: Bool {
+        return _weapon != nil
+    }
+
+    var heroPowerAttack: Int {
+        switch _hero._class {
+        case "MAGE":
+            return 1
```

Comment:
This is not correct, image you have [Fallen Hero](http://www.hearthpwn.com/cards/22268-fallen-hero) in play